controller/comments.go

```diff
@@ -112,10 +113,16 @@ func (c *CommentsController) Delete(ctx *app.DeleteCommentsContext) error {
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, err)
 		}
-		if *identityID != cm.CreatedBy {
```

Comment:
We should actually keep that check. IMO we should allow the comment creators to delete (and update) their own comments even if they are not space collaborators. This is my fault. I missed that in the https://github.com/almighty/almighty-core/issues/1299 (updated now).